pkg/util/mount/mount_helper_windows.go

```diff
@@ -0,0 +1,43 @@
+// +build windows
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mount
+
+import (
+	"k8s.io/klog"
+	"strings"
+)
+
+const (
+	// SMBConnectionError : SMB Connection Error, e.g. password change
+	SMBConnectionError = "password is incorrect"
```

Comment:
I think this message can be localized, is there an error number we can compare against instead? https://docs.microsoft.com/en-us/windows/desktop/Debug/system-error-codes--0-499-
ERROR_INVALID_PASSWORD = 86